ts-nextjs-typeorm/external-content.ts

```diff
@@ -0,0 +1,37 @@
+
+
+import * as fs from "fs/promises"
+import * as express from "express";
+
+export const router = express.Router();
+
+
+router.use(express.static("external-content"))
+// router.use('/external-content', express.static("external-content"))
+
+router.get("/_next/image", async (req, res) => {
+  const file = req.query.url.toString()
+  const content = await fs.readFile(`external-content/${file}`)
+  res.contentType(file)
+  res.send(content)
+})
+
+
+// router.get("/external-content/:file", async (req, res) => {
+//   const file = req.params.file;
+//   const content = await fs.readFile(`external-content/${file}`)
+//   res.contentType(file)
+//   res.send(content)
+// })
+
+export async function nextExternalContentMiddleware(req, res, next) {
+  let file = req.query.url;
+  if (file.startsWith('/external-content')) {
+    file = file.substring(1)
+    const content = await fs.readFile(`${file}`)
+    res.contentType(file)
+    return res.send(content)
+  }
+
+  next()
```

Comment:
does this return automatically? Or if not, does it need to be `await`ed?

Reply:
Ah, I forgot to remove this file.